client/src/components/ReportButton.js

```diff
@@ -0,0 +1,9 @@
+import styled from '@emotion/styled';
+import Button from './Button';
+
+const ReportButton = styled(Button)`
```

Comment:
It's very similar to https://github.com/lmachens/copypasta/pull/30/files#diff-6ab9c5b3fe881baf7b818d382bcbb098